UI.PageModels/Pages/Video/Participant/GetReadyForTheHearingIntroductionPage.cs

```diff
@@ -19,6 +19,12 @@ public CourtRulesPage SkipToCourtRules()
         ClickElement(_skipButton);
         return new CourtRulesPage(Driver, DefaultWaitTime);
     }
+    
+    public bool IsSkipButtonVisible()
```

Comment:
You might want to change this to a wait for element. The component makes an API call which might not return quick enough before this method runs, running a risk this could be flakey.
` WaitForElement(_skipButton, 5)`. If you wan to keep this reason it might be cool to add an overload that allows us to provide a reason in the error thrown